lib/client/index.js

```diff
@@ -388,9 +388,11 @@ client.load = function load (serverSettings, callback) {
     // update brush and focus chart with recent data
     var brushExtent = client.dataExtent();
 
+    chart.theBrush.call(chart.brush)
```

Comment:
based on `chart.theBrush && ` a few lines below it looks like we should check that `chart.theBrush` isn't null here, grouping this with the call bellow in a single if would be better than using `&&` in both places

Reply:
Thanks @jasoncalabrese!  Requested changes are incorporated.